mobile/bindings.go

```diff
@@ -0,0 +1,39 @@
+package lndmobile
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	flags "github.com/jessevdk/go-flags"
+	"github.com/lightningnetwork/lnd"
+)
+
+func Start(extraArgs string, callback Callback) {
+	// Add the extra arguments to os.Args, as that will be parsed during
+	// startup.
+	os.Args = append(os.Args, strings.Fields(extraArgs)...)
```

Comment:
splitting the `extraArgs` string breaks if I want to pass a path that contains a space.
e.g. `/Library/Application\ Support`

Reply:
Can you check if this helps: https://github.com/lightningnetwork/lnd/pull/3282/commits/22f7b0c44ab0bdc2cca7eedcaa73cb4b0386a3e0